static/docker-compose.yml

```diff
@@ -135,6 +135,9 @@ services:
       LOG_LEVEL: "${LOG_LEVEL:-info}"
       REDIS_CONNECTION_STRING: "${REDIS_CONNECTION_STRING:-redis}"
       REDIS_PASSWORD: "${REDIS_PASSWORD:?err}"
+    # Specifically needs the API_INTEGRATION_AUTH_CONFIG_NOTIFICATION_SERVICE generated in src/_configure_plextrac.sh
+    env_file:
+    - .env
```

Comment:
@evanhsu I can't remember how we wrote the engine/sender but we should verify it's only the engine that needs to make api calls.